app/controllers/api/v1/monsters_controller.rb

```diff
@@ -19,9 +21,26 @@ def create
     end
   end
 
-  private
+  def destroy
+    Monster.find(params[:id]).destroy
+    render json: {}, status: :no_content
```

Comment:
We should ideally send something back as a response, even just for us developers to see that the request came back successfully like `{message: "deleted!"}`